html/webappapis/dynamic-markup-insertion/opening-the-input-stream/resources/slow.py

```diff
@@ -0,0 +1,5 @@
+import time
```

Comment:
Maybe best to dedupe this with https://github.com/web-platform-tests/wpt/blob/master/html/semantics/scripting-1/the-script-element/resources/slow.py into /common?=